extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -368,7 +371,10 @@ export class AmpSidebar extends AMP.BaseElement {
       this.historyId_ = -1;
     }
     if (this.openerElement_ && sidebarIsActive && scrollDidNotChange) {
-      tryFocus(this.openerElement_);
+      // As of iOS 12.2, focus() causes undesired scrolling in UIWebViews.
```

Comment:
How about restricting to `const iOSWebview = viewer.isWebviewEmbedded() && platform.isIos();` instead?

(this `isWebviewEmbedded` returns based on whether viewer is setting `webview` param on the URL, if the partner here is not setting that, they probably should, there are a few other places that checks this condition)

Reply:
Good idea, done.

Reply:
Actually, `isWebviewEmbedded()` assumes there's no parent iframe inside the web view. This isn't necessarily true, so I changed it to `isEmbedded()`.